### PR TITLE
Updates to remaining packages/addons/addon-depends

### PR DESCRIPTION
**Pull request by heitbaum**
- blindscan-s2: update to HEAD
- - minor updates over ~ 6 months https://github.com/OpenVisionE2/blindscan-s2/compare/192eaf0b3eccfb9721b3d49b9010b5c1447776e0...master
- ccid: update to 1.4.33
- - changelog:- page down @ https://salsa.debian.org/rousseau/CCID/-/raw/master/README.md
- - update from 1.4.28 to 1.4.33
- - Release 1.4.28 · 11 October 2017
- - Release 1.4.33 · 25 June 2020
- - https://salsa.debian.org/rousseau/CCID/-/commits/master/
- dvb-apps: update sha256 and https pkg_url
- - doco change only
- libconfig: update to 1.7.2 and PKG_SITE
- - update from 1.5 (17 May 2015) to 1.7.2 (6 Jan 2018)
- - https://github.com/hyperrealm/libconfig/releases
- libiconv: update to 1.16
- - update from 1.15 (03 Feb 2017) to 1.16 (2019-04-26)
- - changelog https://git.savannah.gnu.org/gitweb/?p=libiconv.git;a=shortlog;h=refs/tags/v1.16
- - https://git.savannah.gnu.org/gitweb/?p=libiconv.git;a=blob;f=NEWS;h=01ce52e64c4bd67c3cfe42168b202f8dd67803c3;hb=1170b8aa7991fa8c49fb716d6f53c9e0358fceea
- - https://git.savannah.gnu.org/gitweb/?p=libiconv.git;a=blob;f=ChangeLog;h=dcc29f705da7d3be09ff73ecf398c3e78c8d4a97;hb=1170b8aa7991fa8c49fb716d6f53c9e0358fceea
- libugpio: update to 0.0.7
- - update from 0.0.6 (7 Feb 2015 ) to 0.0.7 (29 Nov 2019)
- - https://github.com/mhei/libugpio/releases
- libvncserver: update to 0.9.13
- - update 0.9.11 (2016-12-30) to 0.9.13 (2020-06-13)
- - https://github.com/LibVNC/libvncserver/blob/master/NEWS.md
- libvpx: update to 1.9.0
- - update 1.8.2 (20 Dec 2019) to 1.9.0 (31 Jul 2020)
- - https://chromium.googlesource.com/webm/libvpx/+/6516e974f8c40d0e49b19a4b55b1c98e7432edbb/CHANGELOG
- libwebsockets: update to 4.1.6
- - update from 4.0.7 (5 May 2020) to 4.1.6 (2 Dec 2020)
- - https://github.com/warmcat/libwebsockets
- - https://github.com/warmcat/libwebsockets/blob/main/changelog
- - https://github.com/warmcat/libwebsockets/blob/main/README.md
- - detail all via https://github.com/warmcat/libwebsockets/commits/main
- libzip: update to 1.7.3
- - update 1.6.1 (February 3, 2020) to 1.7.3 (July 15, 2020)
- - https://libzip.org/news/
- miniupnpc: update to 2.2.1
- - update from 2.1 (2018/05/07) to 2.2.1 (20/12/2020)
- - http://miniupnp.free.fr/files/changelog.php?file=miniupnpc-2.2.1.tar.gz
- - http://miniupnp.free.fr/files/
- pcsc-lite: update to 1.9.0
- - update 1.8.23 (18 December 2017) to 1.9.0 (14 June 2020)
- - https://salsa.debian.org/rousseau/PCSC/blob/master/ChangeLog
- protobuf: update to 3.14.0
- - update 3.1 (24 Sep 2016) to 3.14 (14 Nov 2020)
- - https://github.com/protocolbuffers/protobuf/releases
- serdisplib: update to 2.02
- - update 1.97.9 (2010-02-21) to 2.02 (2019-10-27)
- - news - http://serdisplib.sourceforge.net/#news

### CODE REVIEW

File: packages/addons/addon-depends/libvncserver/package.mk [hunk 1]
```diff
@@ -2,14 +2,13 @@
...
-PKG_VERSION="0.9.11"
-PKG_SHA256="193d630372722a532136fd25c5326b2ca1a636cbb8bf9bb115ef869c804d2894"
+PKG_VERSION="0.9.13"
+PKG_SHA256="0ae5bb9175dc0a602fe85c1cf591ac47ee5247b87f2bf164c16b05f87cbfa81a"
 PKG_LICENSE="GPL"
 PKG_SITE="https://libvnc.github.io/"
 PKG_URL="https://github.com/LibVNC/libvncserver/archive/LibVNCServer-$PKG_VERSION.tar.gz"
 PKG_DEPENDS_TARGET="toolchain libjpeg-turbo libpng openssl systemd"
 PKG_LONGDESC="A C library that allow you to easily implement VNC server or client functionality."
-PKG_TOOLCHAIN="autotools"
```
Comment (CvH) [inline packages/addons/addon-depends/libvncserver/package.mk:12]: cmake is used so use cmake options (also faster :) ), just build tested
```
PKG_NAME="libvncserver"
PKG_VERSION="0.9.13"
PKG_SHA256="0ae5bb9175dc0a602fe85c1cf591ac47ee5247b87f2bf164c16b05f87cbfa81a"
PKG_LICENSE="GPL"
PKG_SITE="https://libvnc.github.io/"
PKG_URL="https://github.com/LibVNC/libvncserver/archive/LibVNCServer-$PKG_VERSION.tar.gz"
PKG_DEPENDS_TARGET="toolchain libjpeg-turbo libpng openssl systemd"
PKG_LONGDESC="A C library that allow you to easily implement VNC server or client functionality."

PKG_CMAKE_OPTS_TARGET="-DWITH_GCRYPT=0 \
                       -DWITH_GNUTLS=0 \
                       -DWITH_GTK=0 \
                       -DWITH_SDL=0 \
                       -DWITH_TIGHTVNC_FILETRANSFER=0"
```

Comment (heitbaum) [reply packages/addons/addon-depends/libvncserver/package.mk:12]: Changed

**Review by CvH — APPROVED** [review]

**Comment by CvH** [comment]
```
UNPACK      cxxtools
    APPLY PATCH (common)      packages/addons/addon-depends/cxxtools/patches/cxxtools-2.2-Char_operator_eq_unsigned_int.patch
patching file include/cxxtools/char.h
Hunk #1 succeeded at 130 with fuzz 2 (offset -18 lines).
    APPLY PATCH (common)      packages/addons/addon-depends/cxxtools/patches/cxxtools-2.2-hdstream_stdio.patch
patching file src/hdstream.cpp
    APPLY PATCH (common)      packages/addons/addon-depends/cxxtools/patches/cxxtools-crosscompile.patch
patching file configure
Hunk #1 succeeded at 18511 with fuzz 2 (offset 1195 lines).
can't find file to patch at input line 18
Perhaps you used the wrong -p or --strip option?

```

**Comment by heitbaum** [comment]
Have removed 
- tntnet: update to 3.0
- cxxtools: update to 3.0

Identified dependency on aclocal 1.16.2 which we are not pulling at this time.

**Pull request merged**